src/Pages/List/List.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from "react";
+import ListTag from "./Components/ListTag";
+import ListCategory from "./Components/ListCategory";
+import PopularCreator from "./Components/PopularCreator";
+import { AiFillCaretDown } from "react-icons/ai";
+import "./List.scss";
+
+class List extends Component {
+  constructor() {
+    super();
+    this.state = {
+      listName: "사진",
+      bannerBgSrc:
+        "https://images.unsplash.com/photo-1536924940846-227afb31e2a5?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1047&q=80",
+      categoryToggle: false,
+      category: [],
+      listBannerTags: [],
+      popularCreator: [],
+    };
+  }
+
+  componentDidMount() {
+    // 상단 배너 카테고리
+    fetch("http://localhost:3000/Data/List/CATEGORY.json", {
```

Comment:
fetch 잘해주셨네요~~! 아래 내용을 반영해서 수정해주시면 더 좋은 코드가 될 것 같습니다. 
- get method는 아래와 같이 생략 가능합니다. 
- API 주소들도 config.js 파일에 담아 관리하시면 api주소가 바뀔 때 config파일에서 편리하게 수정할 수 있기 때문에 유지보수에 더 용이합니다 :) config.js 파일에서 변수에 선언해 주신 뒤에 import 해서 사용해주세요!

```suggestion
    fetch(${ListAPI})
```

Reply:
/Data 앞부분을 변수로 선언해서 사용했습니다!